packages/playground/src/main.ts

```diff
@@ -10,7 +10,7 @@ const room = params.get('room') ?? '';
 window.onload = () => {
   const editor = createEditor({
     room,
-    providers: [DebugProvider],
+    providers: [IndexDBProvider, DebugProvider],
   });
```

Comment:
Change: Separate out another playground entry page with just IndexDB to ensure that CI tests work the same and we can test individual configurations separately.